lib/core/src/Cardano/Wallet/Api/Types.hs

```diff
@@ -334,6 +341,18 @@ newtype ApiFee = ApiFee
     { amount :: (Quantity "lovelace" Natural)
     } deriving (Eq, Generic, Show)
 
+data ApiEpochNumber = ApiEpochNumberLatest | ApiEpochNumber EpochNo
+    deriving (Eq, Generic, Show)
+
+data ApiNetworkParameters = ApiNetworkParameters
+    { genesisBlockHash :: !(ApiT (Hash "Genesis"))
+    , blockchainStartTime :: !(ApiT StartTime)
+    , slotLength :: !(Quantity "second" NominalDiffTime)
+    , epochLength :: !(Quantity "slot" Word32)
+    , epochStability :: !(Quantity "block" Word32)
+    , activeSlotCoefficient :: !(Quantity "percent" Percentage)
```

Comment:
> Percentage

Though I think you might find in the next PR that activeSlotCoeff is a `Double` whereas `Percentage` is a `Word, [0, 100]`

Reply:
Ok, U/S says:
> active_slot_coefficient: *percentage

And in `x-percentage` is a `number`. So it should be able to be a floating point.

I imagine some users would like to have the full `33.333333%` instead of just `33%`.